Tests/common_BioSQL.py

```diff
@@ -701,7 +701,7 @@ def test_del_db_items(self):
 
             del db[seq_id]
             # check to see that the entry in the bioentry table is removed
-            self.assertEqual(seq_id in db, False)
+            self.assertFalse(seq_id in db)
```

Comment:
The original was strange, but this can be ``self.assertNotIn(seq_id, db)``

https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertNotIn